src/tree/AttachedAccountsTreeItem.ts

```diff
@@ -107,12 +111,24 @@ export class AttachedAccountsTreeItem extends AzureParentTreeItem {
 
         const attachedAccounts: AzureTreeItem[] = await this.getAttachedAccounts();
 
-        return attachedAccounts.length > 0 ? attachedAccounts : [new GenericTreeItem(this, {
-            contextValue: 'cosmosDBAttachDatabaseAccount',
-            label: 'Attach Database Account...',
-            commandId: 'cosmosDB.attachDatabaseAccount',
-            includeInTreeItemPicker: true
-        })];
+        if (attachedAccounts.length > 0) {
+            return attachedAccounts;
+        } else {
+            const genericTreeItemWithoutEmulator = new GenericTreeItem(this, {
+                contextValue: 'cosmosDBAttachDatabaseAccount',
+                label: 'Attach Database Account...',
+                commandId: 'cosmosDB.attachDatabaseAccount',
+                includeInTreeItemPicker: true
+            });
+            const genericTreeItemWithEmulator = new GenericTreeItem(this, {
+                contextValue: 'cosmosDBAttachDatabaseAccount',
+                label: 'Attach Emulator...',
+                commandId: 'cosmosDB.attachEmulator',
+                includeInTreeItemPicker: true
+            });
+            return AttachedAccountsTreeItem.isPlatformWindows() ? [genericTreeItemWithoutEmulator, genericTreeItemWithEmulator] :
+                [genericTreeItemWithoutEmulator];
```

Comment:
nit: Minor whitespace change 
```suggestion
            return AttachedAccountsTreeItem.isPlatformWindows() ? 
                [genericTreeItemWithoutEmulator, genericTreeItemWithEmulator] :
                [genericTreeItemWithoutEmulator];
```